pkgs/applications/science/misc/openmodelica/mkderivation/src-main.nix

```
@@ -0,0 +1,7 @@
+{
+    url = "https://github.com/OpenModelica/OpenModelica/";
+    rev = "08fd3f9144235f209a4ed7602bfadb32b1823628";
+    sha256 = "0clgqk9ilnr43iyl5sdzwfzqpnw9amfy1npdgkpgm1wfnsvz6xrw";
+    fetchSubmodules = true;
```

```suggestion
  url = "https://github.com/OpenModelica/OpenModelica/";
  rev = "08fd3f9144235f209a4ed7602bfadb32b1823628";
  sha256 = "0clgqk9ilnr43iyl5sdzwfzqpnw9amfy1npdgkpgm1wfnsvz6xrw";
  fetchSubmodules = true;
```

Why is this not using fetchFromGitHub?

---

fetchFromGitHub downloads zip's without submodules.